src/__tests__/store/message/reducer.test.ts

```diff
@@ -0,0 +1,74 @@
+import { messageActionTypes, IMessageState } from "../../../src/store/message/types";
```

Comment:
imports not correctly finding files in path. fix paths in all import statements.